content/200-concepts/300-more/050-environment-variables/100-managing-env-files-and-setting-variables.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: 'Managing .env files and setting variables'
+metaTitle: 'Managing .env files and setting variables'
+metaDescription: 'Learn how to manage .env files and set environment variables'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Although [Prisma creates an `.env` file for you upon installation](./#how-does-prisma-use-environment-variables), there can be times when you want to:
+
+- Bypass `.env` files and let Prisma use the system environment variables directly
+- Use `.env` files from a location that the Prisma CLI does not check [by default](./#using-env-files)
```

Comment:
Where does this link to?

Reply:
This links to https://www.prisma.io/docs/concepts/more/environment-variables#using-env-files. `./` indicates the index file at the root of the folder, `#using-en-files` is a heading in the index file.